integrated_channels/moodle/models.py

```diff
@@ -82,6 +101,15 @@ class MoodleEnterpriseCustomerConfiguration(EnterpriseCustomerPluginConfiguratio
         )
     )
 
+    decrypted_token = EncryptedCharField(
+        max_length=255,
+        verbose_name="Encrypted Webservice Token",
+        blank=True,
+        help_text=_(
+            "The encrypted API user's token used to obtain new tokens."),
+        null=True,
+    )
```

Comment:
Just to clear out the confusion with encrypted/decrypted data, can we add this line in help_text on all newly added fields? "It will be encrypted when stored in the database."